.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -41,10 +41,7 @@ body:
         - "avm/ptn/authorization/policy-assignment"
         - "avm/ptn/authorization/resource-role-assignment"
         - "avm/ptn/authorization/role-assignment"
-        # - "avm/ptn/avd-lza/insights"
```

Comment:
No need to remove comments, they are expected to be onboarded (currently module proposals)

Reply:
The pipeline was complaining. That's why I removed them.